trie/utils/verkle.go

```diff
@@ -277,7 +277,7 @@ func GetTreeKeyStorageSlotWithEvaluatedAddress(evaluated *verkle.Point, storageK
 	} else {
 		pos.Add(mainStorageOffsetBig, pos)
 	}
-	treeIndex, overflow := uint256.FromBig(pos.Div(pos, verkleNodeWidthBig))
+	treeIndex, overflow := uint256.FromBig(big.NewInt(0).Div(pos, verkleNodeWidthBig))
```

Comment:
This was touching `pos` storing the `Div` operation. This is wrong since it's basically wiping the last 8 bits that we need for the "`Mod`" below.

Reply:
aaaah right good catch